apps/editors/tests/test_views.py

```diff
@@ -1413,6 +1413,13 @@ def search(self, *args, **kw):
         return r
 
 
+def login_as_admin(fn):
```

Comment:
Thinking about it, why not, but it doesn't really spare us any code, right? It's a decorator here, instead of having a one line in each test (but it's offset by the line added to decorate the test).